refresh/templates/extensions/PushNotificationsExtension.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import BluemixPushNotifications
+import CloudFoundryConfig
+
+extension PushNotifications {
+
+   public convenience init(service: PushNotificationsService) {
+
+        let regionVar = "<%- service.region %>"
+        var region: String
+
+        regionVar.isEmpty ? (region = PushNotifications.Region.US_SOUTH) : (region = regionVar)
+
+        self.init(
+            bluemixRegion: region,
+            bluemixAppGuid: "<%- service.guid %>", 
+            bluemixAppSecret: "<%- service.secret %>" )
```

Comment:
@tunniclm I am re-working the extension and this is no longer ready for review, I'm going to close the PR and re-open when it's ready.